### DISCUSSION TITLE
Using google recaptcha in China

### DISCUSSION THREAD

**Discussion by chrispeeters** [lukeraymonddowning/honey#46] [Q&A]
Links pointing to anything "google.com" are blocked in mainland China. Reading up on this, I found that changing the servers from www.google.com to www.rechapta.net would resolve that issue.

I found two files in the vendor directory pointing to google.com:
```
/vendor/lukeraymonddowning/honey/src/Captcha/Recaptcha.php
/vendor/lukeraymonddowning/honey/src/Views/Recaptcha.php
```
Is there a way to override the reference to google.com in those files?

I have correspondents in China that could help me test such a solution.

**Comment by lukeraymonddowning** [comment]
Howdy! Atm there is no option, but would accept a PR that changes the URLs 👍

**Reply by chrispeeters** [reply]
Let me first try out the actual scenario and and test it from within China, then I'll sort the PR stuff out.
